pennylane/measure.py

```diff
@@ -18,14 +18,8 @@
 and measurement samples.
 """
 import pennylane as qml
-from .operation import Observable, Sample, Variance, Expectation, Probability, Tensor
-from .qnodes import QuantumFunctionError
-
-
-def _remove_if_in_queue(op):
-    r"""Helper function to handle removing ops from the QNode queue"""
-    if op in qml._current_context.queue:
-        qml._current_context.queue.remove(op)
```

Comment:
Nice to finally be able to remove this!

Reply:
"remove if in queue" is now the default behaviour of `QueuingContext.remove_operation`.